### PR TITLE
Update buildpacks to include `jq`.

### PR DESCRIPTION
**Pull request by aarlt**
The asm json import/export use `jq` to compare the output (see https://github.com/ethereum/solidity/pull/12704).

### CODE REVIEW

File: scripts/docker/buildpack-deps/Dockerfile.ubuntu1604.clang.ossfuzz [hunk 1]
```diff
@@ -32,14 +32,14 @@ RUN apt-get update; \
...
-# Install cmake 3.21.2 (minimum requirement is cmake 3.10)
-RUN wget https://github.com/Kitware/CMake/releases/download/v3.21.2/cmake-3.21.2-Linux-x86_64.sh; \
-    test "$(sha256sum cmake-3.21.2-Linux-x86_64.sh)" = "3310362c6fe4d4b2dc00823835f3d4a7171bbd73deb7d059738494761f1c908c  cmake-3.21.2-Linux-x86_64.sh"; \
-    chmod +x cmake-3.21.2-Linux-x86_64.sh; \
-    ./cmake-3.21.2-Linux-x86_64.sh --skip-license --prefix="/usr"
+# Install cmake 3.22.2 (minimum requirement is cmake 3.10)
+RUN wget https://github.com/Kitware/CMake/releases/download/v3.22.2/cmake-3.22.2-linux-x86_64.sh; \
+    test "$(sha256sum cmake-3.22.2-linux-x86_64.sh)" = "faaa0465b72fe467fa193954c9bb67141048a2c7acc3236975ac20fdf241582e  cmake-3.22.2-linux-x86_64.sh"; \
+    chmod +x cmake-3.22.2-linux-x86_64.sh; \
+    ./cmake-3.22.2-linux-x86_64.sh --skip-license --prefix="/usr"
```
Comment (aarlt) [inline scripts/docker/buildpack-deps/Dockerfile.ubuntu1604.clang.ossfuzz]: For `ubuntu1604.clang.ossfuzz` there is already a cmake installed in `/usr/local/bin/cmake` v3.21.1, where `PATH` is set to `/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/out` - so the "old" version has higher priorities. Maybe we could just change the `prefix` to install the new version in `/usr/local`.

Comment (aarlt) [reply scripts/docker/buildpack-deps/Dockerfile.ubuntu1604.clang.ossfuzz]: I will now change cmake installation `prefix` to `/usr/local`.

File: scripts/docker/buildpack-deps/Dockerfile.ubuntu2004 [hunk 2]
```diff
@@ -37,12 +37,18 @@ RUN set -ex; \
...
 		; \
 	apt-get install -qy python3-pip python3-sphinx; \
 	pip3 install codecov; \
 	rm -rf /var/lib/apt/lists/*
 
+# Install cmake 3.22.2 (minimum requirement is cmake 3.10)
+RUN wget https://github.com/Kitware/CMake/releases/download/v3.22.2/cmake-3.22.2-linux-x86_64.sh; \
+    test "$(sha256sum cmake-3.22.2-linux-x86_64.sh)" = "faaa0465b72fe467fa193954c9bb67141048a2c7acc3236975ac20fdf241582e  cmake-3.22.2-linux-x86_64.sh"; \
+    chmod +x cmake-3.22.2-linux-x86_64.sh; \
+    ./cmake-3.22.2-linux-x86_64.sh --skip-license --prefix="/usr/local"
```
Comment (ekpyron) [inline scripts/docker/buildpack-deps/Dockerfile.ubuntu2004]: This doesn't seem like a good idea - we shouldn't be testing against a customly installed version of ``cmake``.
If we want to test builds against newer versions of ``cmake`` than what is packaged in the base images (resp. available for ubuntu LTS), we can use archlinux build runs instead.

Comment (aarlt) [reply scripts/docker/buildpack-deps/Dockerfile.ubuntu2004]: Ah ok! Basically we don't want to change the build tools contained in the default ubuntu distribution. Yeah that makes sense. I will checkout the archlinux build.

**Review by ekpyron — APPROVED** [review]

**Pull request merged**